lib/perl5/site_perl/ReadToolsUtils.pm

```diff
@@ -64,7 +64,7 @@ Output:
 =cut
 sub get_trimming_args {
 	my ($qualThreshold, $minLength, $discardRemainingNs, $noTrimQuality, $no5ptrim) = @_;
-	my $trim_args = " --readFilter ReadLengthReadFilter --minReadLength $minLength";
+	my $trim_args = " --readFilter ReadLengthReadFilter --minReadLength $minLength --maxReadLength 1000";
```

Comment:
@robmaz - actually I think that the maximum read lenght should be an opt-in for the new version of ReadTools. The main problem is that the read-length filter is inherit from GATK, and it is true that it should be required (at least one of min/max read length). They decided to have the max as the required one, and the min set by default to 1 - which makes sense, but not for the case of the distmap pipeline...